src/lsp-server.ts

```diff
@@ -655,14 +656,15 @@ export class LspServer {
                 offset: params.position.character + 1,
                 triggerCharacter: getCompletionTriggerCharacter(params.context?.triggerCharacter),
                 triggerKind: params.context?.triggerKind,
+                includeInsertTextCompletions: true,
             }));
             const { body } = result;
             const completions: lsp.CompletionItem[] = [];
             for (const entry of body?.entries ?? []) {
                 if (entry.kind === 'warning') {
                     continue;
                 }
-                const completion = asCompletionItem(entry, file, params.position, document, this.features);
+                const completion = asCompletionItem(entry, file, params.position, document, this.features, body);
```

Comment:
It's weird to pass just a single `entry` as one argument and then also the whole response as another. I'd prefer that you just pass `body.optionalReplacementSpan`. And you can move the argument after `entry` maybe since it kind of belongs better there.